k3d: add page (#9273)
tldr-pages/tldr: pages/common/k3d.md

# k3d

> > Wrapper CLI to easily create k3s clusters inside docker.

Reviewer: ```suggestion
> Wrapper CLI to easily create k3s clusters inside Docker.
```

> > More information: <https://k3d.io/>.

Reviewer: ```suggestion
> More information: <https://k3d.io>.
```

- Create a cluster:

`k3d cluster create {{cluster_name}}`

- Delete a cluster:

`k3d cluster delete {{cluster_name}}`

- Create a new containerized k3s node:

`k3d node create {{node_name}}`

> - Import an image from docker into k3d [c]luster:

Reviewer: ```suggestion
- Import an image from Docker into a k3d cluster:
```

> - Import a image from docker into k3d cluster:

Reviewer 2: ```suggestion
- Import an image from docker into k3d [c]luster:
```
We prefer long arguments but when we use short single-character arguments we use mnemonics ([]) to show the long arguments.

> `k3d image import {{image_name}} -c {{cluster_name}}`

Reviewer: We prefer long arguments if available
```suggestion
`k3d image import {{image_name}} --cluster {{cluster_name}}`
```
